src/components/organizations/index.jsx

```diff
@@ -29,10 +84,31 @@ const Organizations = () => {
 
   return (
     <Box p='2.25rem 5rem' flexGrow={1} overflow='auto'>
-      <Typography fontSize='1.5rem' color={gray700} fontWeight={600} mb='1.5rem'>
-        {organizations.length} Organizations
-      </Typography>
+      <Stack direction="row" sx={{ justifyContent: "space-between" }}>
+        <Typography fontSize='1.5rem' color={gray700} fontWeight={600} mb='1.5rem'>
+          {organizations?.length} Organizations
+        </Typography>
+        <Button type="string" startIcon={<GroupAddOutlinedIcon />} onClick={createOrganization}>Create a new organizations</Button>
```

Comment:
typo, change organizations with organization, we are trying to create one, not many